src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -192,381 +191,387 @@ the MIT License. See LICENSE in the project root for license information. -->
                                                Margin="10,0,0,0"/>
-                                </Grid>
-                            </StackPanel>
-                        </ContentPresenter>
+                                    </Grid>
+                                </StackPanel>
+                            </ContentPresenter>
 
-                        <!--Retro Terminal Effect-->
-                        <ContentPresenter Style="{StaticResource SettingContainerStyle}">
-                            <CheckBox x:Uid="Profile_RetroTerminalEffect"
+                            <!--Retro Terminal Effect-->
+                            <ContentPresenter Style="{StaticResource SettingContainerStyle}">
+                                <CheckBox x:Uid="Profile_RetroTerminalEffect"
                                       IsChecked="{x:Bind State.Profile.RetroTerminalEffect, Mode=TwoWay}"
                                       Style="{StaticResource CheckBoxSettingStyle}"/>
-                        </ContentPresenter>
+                            </ContentPresenter>
+                        </StackPanel>
 
                         <!--Grouping: Cursor-->
-                        <TextBlock x:Uid="Profile_CursorHeader" Style="{StaticResource GroupingHeader}"/>
-
-                        <!--Cursor Shape-->
-                        <ContentPresenter Style="{StaticResource SettingContainerStyle}">
-                            <muxc:RadioButtons x:Uid="Profile_CursorShape"
-                                               ItemsSource="{x:Bind CursorShapeList, Mode=OneWay}"
-                                               SelectedItem="{x:Bind CurrentCursorShape, Mode=TwoWay}"
-                                               ItemTemplate="{StaticResource EnumRadioButtonTemplate}"
-                                               SelectionChanged="CursorShape_Changed"
-                                               Style="{StaticResource RadioButtonsSettingStyle}"/>
-                        </ContentPresenter>
-
-                        <!--Cursor Height-->
-                        <ContentPresenter Style="{StaticResource SettingContainerStyle}"
-                                          Visibility="{x:Bind IsVintageCursor, Mode=OneWay}">
-                            <muxc:NumberBox x:Uid="Profile_CursorHeight"
-                                            Value="{x:Bind State.Profile.CursorHeight, Mode=TwoWay}"
-                                            Style="{StaticResource NumberBoxSettingStyle}"
-                                            SmallChange="1"
-                                            LargeChange="10"/>
-                        </ContentPresenter>
+                        <StackPanel Style="{StaticResource PivotStackStyle}">
+                            <TextBlock x:Uid="Profile_CursorHeader" Style="{StaticResource SubtitleTextBlockStyle}"/>
+
+                            <!--Cursor Shape-->
+                            <ContentPresenter>

```

Comment:
Again, if you're using a `ContentPresenter` without styling, I suggest styling it, then overriding the margin plz.